keps/sig-instrumentation/2305-metrics-cardinality-enforcement/README.md

```diff
@@ -236,19 +264,145 @@ This would then be interpreted by our machinery as this:
 
 ```
 
-## Open-Question
-
+## Design Details
+### Test Plan
+Unit test to verify that the metric label will be set to "unexpected" if the metric encounters label values outside our explicit allowlist of values.
+### Graduation Criteria
+The allowlist of metrics can be configured via the exposed flag and the unit test is passed.
+### Upgrade / Downgrade strategy
+N/A
+### Version Skew Strategy
+N/A
+
+## Production Readiness Review Questionnaire
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [ ] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name:
+    - Components depending on the feature gate:
+  - [x] Other
+    - Describe the mechanism: The metric cardinality will be enforced by default. The allowlist of label values for a metric is dynamically configured with flag.
```

Comment:
I don't think we can just default this to on in alpha. This should be behind a feature flag so it can be turned off.